### ISSUE TITLE
Microsoft Edge support

### ISSUE THREAD

**Issue by PxlBuzzard** [gabrielecirulli/hn-special#100]
According to [this article](https://developer.microsoft.com/en-us/microsoft-edge/platform/documentation/extensions/), it should be trivial to convert an extension that works in Chrome over to Edge.

**Comment by aisk** [comment]
Since Edge will be based on chrome, I think this will not be an issue?

**Comment by PxlBuzzard** [comment]
Correct, it should work great in Chromium-based Edge. The only thing left to figure out would be whether we can officially publish it in Microsoft's extension store versus having people install it through Chrome Web Store.
